packages/plugins/Gfi/src/components/Gfi.vue

```diff
@@ -135,4 +146,11 @@ export default Vue.extend({
     z-index: 1;
   }
 }
+
+.polar-plugin-gfi-move-handle {
+  left: 0;
+  right: 0;
+  overflow-y: unset;
```

Comment:
```suggestion
  overflow-y: visible;
```

Reply:
Is `unset` producing a bug?

Reply:
After fiddling around a bit it seems like the parameter is not necessary whatsoever 🤔 Can you highlight the usecase?

Reply:
It prevents a vertical scrollbar on the element. You may e.g. open a GFI with huge picture to see that it will, else, contain a vertical scrollbar. MoveHandle is not supposed to do that, but to be draggable up until the contents bottom is reached. (See sketches in internal EPIC.) However, since `9999` has been changed to `maxMobileHeight`, the GFI is currently stuck at the screen's top.

Reply:
Removing the value here should be the way either way, as the changes in `feature/icon_menu_move_handle` handle that issue.  
As the FeatureList is the first feature in the MoveHandle I am currently looking at the specific use cases to have it working properly as well.